daemon/daemon.go

```diff
@@ -0,0 +1,315 @@
+package daemon
```

Comment:
Does this tool really needs to be a daemon?
This requires many additional helper dependencies and many difficult to test code.